README.md

```diff
@@ -82,7 +82,7 @@ npm install --save @bigcommerce/checkout-sdk
 
 You can also use this library by referencing a CDN URL.
 
-```
+```http
```

Comment:
why do we need it here?

Reply:
It's optional, but my MD linter picked it up when editing so I included.
 https://github.com/updownpress/markdown-lint/blob/master/rules/040-fenced-code-language.md